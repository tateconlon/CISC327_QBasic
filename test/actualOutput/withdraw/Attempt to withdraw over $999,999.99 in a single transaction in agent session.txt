login as agent successful
Account 1234567 does not exist
transaction code 99999999 invalid
Account 1234567 does not exist
transaction code 99999999 invalid
Account 1234567 does not exist
transaction code 100000000 invalid
Transaction summary written and system logged out successfully