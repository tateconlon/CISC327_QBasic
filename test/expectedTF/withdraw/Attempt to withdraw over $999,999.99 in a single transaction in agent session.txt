DEP 1234567 99999999 *** ***
DEP 1234567 99999999 *** ***
EOS