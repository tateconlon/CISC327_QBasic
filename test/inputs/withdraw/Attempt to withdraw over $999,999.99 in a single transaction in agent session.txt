login
agent
deposit
1234567
99999999
deposit
1234567
99999999
withdraw
1234567
100000000
logout