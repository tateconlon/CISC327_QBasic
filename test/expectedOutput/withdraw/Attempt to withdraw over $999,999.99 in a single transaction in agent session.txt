login as agent successful
Cannot withdraw more than $999,999.99 in a single transaction in agent mode
Transaction summary written and system logged out successfully